Soupsieve is not available in Solaris and not necessary
for bs4 to run (only a CSS selector is not available).

--- beautifulsoup4-4.11.1/setup.py
+++ beautifulsoup4-4.11.1/setup.py
@@ -18,9 +18,7 @@ setup(
     download_url = "https://www.crummy.com/software/BeautifulSoup/bs4/download/",
     description="Screen-scraping library",
     python_requires='>=3.6.0',
-    install_requires=[
-        "soupsieve >1.2",
-    ],
+    install_requires=[],
     tests_require=['pytest'],
     long_description=long_description,
     long_description_content_type="text/markdown",
--- beautifulsoup4-4.11.1/beautifulsoup4.egg-info/requires.txt
+++ beautifulsoup4-4.11.1/beautifulsoup4.egg-info/requires.txt
@@ -1,5 +1,3 @@
-soupsieve>1.2
-
 [html5lib]
 html5lib
 
